utilities.py

```diff
@@ -128,7 +128,7 @@ def getNetworkProcessInfo(self,programName):
         status = self.checkProgramStatus(programName)
         pid = ""
         if status [ 0 ]:
-            pid = ",".join(status[1])
+            pid = ",".join(str(p).strip("b'") for p in status[1])
 
         txt = self._PROCESSID + " = " + pid
```

Comment:
If you need to convert this to `str`, why not do it on L133?

Reply:
I agree, this is the one place the array of PID should be converted to type `str`, but using `str()` and stripping `b'` is just very strange.  Use `.decode(errors='ignore')`, given that the only purpose in this method is to prepare text for display on the user interface.